src/cli/domain/init-template/install-template.js

```diff
@@ -16,29 +15,27 @@ module.exports = function installTemplate(options, callback) {
     templateType
   } = options;
 
-  logger.log(strings.messages.cli.installCompiler(compiler));
+  const npmOptions = {
+    dependency: compiler,
+    installPath: componentPath,
+    isDev: true,
+    save: true
+  };
 
-  const args = ['install', '--save-dev', '--save-exact', compiler];
-  const installProc = spawn('npm', args, {
-    cwd: componentPath,
-    stdio: 'inherit'
-  });
+  logger.log(strings.messages.cli.installCompiler(compiler));
 
-  installProc.on('error', () => callback('template type not valid'));
-  installProc.on('close', code => {
-    if (code !== 0) {
-      return callback('template type not valid');
+  npm.installDependency(npmOptions, (err, result) => {
+    const errorMessage = 'template type not valid';
+    if (err) {
+      return callback(errorMessage);
     }
-    const maybeCompiler = require(path.join(
-      componentPath,
-      'node_modules',
-      compiler
-    ));
-
-    if (!isValidTemplate(maybeCompiler, { compiler: true })) {
-      return callback('template type not valid');
+
+    const installedCompiler = tryRequire(result.dest);
+
+    if (!isValidTemplate(compiler, { compiler: true })) {
```

Comment:
Ah! that should have been `isValidTemplate(installedCompiler...` instead! Bugfix coming in another PR: https://github.com/opentable/oc/pull/694